constraints.txt

```diff
@@ -0,0 +1,7 @@
+click==7.1.2
```

Comment:
this doesn't look like a constraints file. This should be generated using `pip-tools`. Please add this [script](https://github.com/OsaroAI/osaro-default-python-template/blob/main/bin/upgrade_constraints.sh) to the repo and generate constraints by running it.